util/file_system.go

```diff
@@ -73,9 +74,18 @@ func ValidateFilePath(path string) (fs.FileInfo, error) {
 	return fileInfo, nil
 }
 
-func CheckIfRunningInRootDirectory(path string) {
-	_, err := os.Stat(computePath(path, "rules/"))
-	if err != nil && os.IsNotExist(err) {
-		fmt.Println("WARNING: The command must point at a folder that contains the package for the rules.\n	 If the rules were generated using the template command, make sure you have the\n	 /rules and /lib folder in your current running directory or provide an optional path argument pointing to that location.")
+func CheckIfRunningInRootDirectory(paths []string) {
```

Comment:
Albeit the fact that the function existed beforehand, could its name be changed to explain better what it does? 

The name suggests we're checking if the current working directory is the root of this repository.

However, it seems the function validates the existence of the `rules` directory in a depth of 2 levels among any of the provided paths.

Perhaps a name similar to `CheckIfRulesExistInPaths` or anything to that effect would be more descriptive?

Reply:
Perhaps a test should be added for this function?